tests/conftest.py

```diff
@@ -7556,21 +7552,22 @@ def _multibatch_generic_csv_generator(
             5: "category5",
             6: "category6",
         }
-        for batch_num in range(num_event_batches):
+        for batch_num in range(num_event_batches):  # type: ignore[arg-type]
```

Comment:
I'm guessing reording the params and making the `num_` params non-optional is a larger lift? Might be cleanest to just assert instance on them at the top of the method. But that's totally non-blocking.